lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -543,6 +586,7 @@ class TraceProcessor {
         firstContentfulPaint: frameTimings.timings.firstContentfulPaint,
         firstMeaningfulPaint: frameTimings.timings.firstMeaningfulPaint,
         largestContentfulPaint: frameTimings.timings.largestContentfulPaint,
+        largestContentfulPaintAllFrames: maybeGetTiming(lcpAllFramesEvt && lcpAllFramesEvt.ts),
```

Comment:
something i spotted while looking at the trace event..... 

https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/page_load_metrics/observers/core/ukm_page_load_metrics_observer.cc;l=1091-1095;drc=2b934611e4e480b7121488288f97ef27e0950e2a

It seems like sometimes the trace event carries a specific timestamp.. so i think we'd want to use that rather than the event's `ts`.

Reply:
I can't seem to find the timestamp you're referring to. There is only a `durationInMilliseconds` property.

Reply:
my silly project agrees with @adamraine https://github.com/connorjclark/chrome-trace-events-tsc/blob/master/dist/lh-trace-events.d.ts#L487 (altho I only used one recent trace from cnn.com)

Reply:
@connorjclark thanks for checking. yeah you can see in the .cc source i linked that there are two ways the LCP trace event are emitted. I haven't really tracked back the conditions that determine which of the two are used..  but.. we'd need enough traces to get both of em for yr project to catch it, i guess.  no big deal though.


@adamraine ah yeah.. actually the `durationInMilliseconds` prop is exactly what i was talking about. (I hadn't clicked through on DataAsTraceValue() till now).

But yes it looks vaguely like: if durationInMilliseconds is being provided, it should be used rather than the `ts` of the event. @npm1 is that what's going on?

Reply:
> if durationInMilliseconds is being provided, it should be used rather than the ts of the event.

👀 at our time origin complexities, oh boy 😅

Reply:
Correct, `durationInMilliseconds` is the LCP value. Sorry for the confusion, maybe there was a way to have the trace event's timestamp be the LCP itself? It's probably a bit late to fix.